panel/src/config/fiber.js

```diff
@@ -56,43 +63,57 @@ const Fiber = {
     );
   },
 
-  component(name) {
-    return Vue.component(name);
+  /**
+   * Prepares a set of values to
+   * be included in a comma-separated list
+   * in a URL query (see include & only params)
+   *
+   * @param {string|Array} array
+   * @returns Array
+   */
+  arrayToString(array) {
+    if (Array.isArray(array) === false) {
+      array = [array];
```

Comment:
`array.join()` on an array with a single value is just the value converted to string:

```suggestion
      return String(array);
```